src/utils/GitHub.js

```diff
@@ -1,7 +1,7 @@
 import {Octokit} from '@octokit/rest'
 import PkgJson from '../../package.json'
 import debug from './debug'
-import {isRunningLocally} from './network'
+// import {isRunningLocally} from './network'
```

Comment:
revert

Reply:
I do think we want this check for running local.. the problem is that if you use live comments from GitHub when developing you can quickly hit per-user quota limits.  I added this to use the test comments when developing and then flip them when I ready to test with more realistic data.